### DISCUSSION TITLE
Store transaction nVersion as uint32 t july 25 2024

### DISCUSSION THREAD

**Discussion by VirdiJatin** [Bitshala/BitcoinCore-PR-Review-Club#114] [PR Reviews]

**Comment by VirdiJatin** [comment]
Hey @paperpsych Ping for thumbnail..

**Comment by paperpsych** [comment]
@VirdiJatin Need more of the details... like the hashtags and PR number !!

**Comment by paperpsych** [comment]
@VirdiJatin 
![Bitshala RC (250724)](https://github.com/user-attachments/assets/10a13dfb-3326-487d-b7b8-6fb613ac64f4)
